scripts/exregional_run_post.sh

```diff
@@ -243,7 +243,7 @@ cp_vrfy ${EXECDIR}/upp.x .
 if [ ${PREDEF_GRID_NAME} = "RRFS_CONUS_3km_HRRRIC" ]; then
   grid_specs_rrfs="lambert:-97.5:38.500000 237.826355:1746:3000 21.885885:1014:3000"
 elif [ ${PREDEF_GRID_NAME} = "RRFS_CONUS_3km" ]; then
-  grid_specs_rrfs="lambert:-97.5:38.500000 237.280472:1799:3000 21.138123:1059:3000"
+  grid_specs_rrfs="lambert:-97.5:38.500000 237.280472:1799:3000 21.138115:1059:3000"
```

Comment:
@EricJames-NOAA Could you elaborate more on what triggered this number change? Thanks!

Reply:
@guoqing-noaa sorry for the late response.  I was just double checking the grid specs for both the CONUS 3km and NA 3km RRFS domains, and found this small discrepancy.  I found the correct values by using wgrib2 -V on a recent RRFS-B output file.  I'm not sure when the domain changed, or if this was an error from a long time ago.